.github/ISSUE_TEMPLATE/issue-request-promotion-graduated.yaml

```diff
@@ -0,0 +1,63 @@
+name: KIT-Promotion-Graduated
+description: Request to promote your KIT
+title: "[Promote KIT To Graduated]: "
+labels: ["kit-promotion"]
+assignees:
+  - octocat
```

Comment:
I would leave this out if it is not clear who will be responsible @octocat seems for me to generic

Reply:
hi @LuisRickert also your could reference it and make it as resolved 😉 
[chore: remove octocat as asignee, remove unneccessary information](https://github.com/eclipse-tractusx/sig-project-managment/pull/6/commits/e2ad74d20343f14dabadc8ce903cc8aaa0b6f805)